src/plugins/dashboard/public/application/top_nav/editor_menu.tsx

```diff
@@ -302,11 +308,7 @@ export const EditorMenu = ({ dashboardContainer, createNewVisType }: Props) => {
         <EuiContextMenu
           initialPanelId={0}
           panels={getEditorMenuPanels(closePopover)}
-          className={`dshSolutionToolbar__editorContextMenu ${
-            IS_DARK_THEME
-              ? 'dshSolutionToolbar__editorContextMenu--dark'
-              : 'dshSolutionToolbar__editorContextMenu--light'
-          }`}
+          className={`dshSolutionToolbar__editorContextMenu`}
```

Comment:
The extra classes aren't necessary, as was described in https://github.com/elastic/kibana/pull/139145#discussion_r966383527